config/application.yml.default

```diff
@@ -158,6 +158,42 @@ lexisnexis_threatmetrix_timeout: 1.0
 lexisnexis_threatmetrix_enabled: false
 lexisnexis_threatmetrix_mock_enabled: true
 lexisnexis_threatmetrix_required_to_verify: false
+lexisnexis_threatmetrix_js_signing_cert: |
```

Comment:
Cert contains a public key used verify signature on JS from ThreatMetrix.

Reply:
is this the "real" public cert?

Reply:
Yes, I'm not sure if the best way to handle this would be to check it into the repo somewhere or have it as a config?

Reply:
I'd prefer we keep "real" certs out of source code, and have them in our config only